client/src/pages/Landing/index.tsx

```diff
@@ -1,93 +1,185 @@
-import AppBar from '@material-ui/core/AppBar';
-import AppToolbar from '@material-ui/core/Toolbar';
-import GithubIcon from '@material-ui/icons/GitHub';
-import { faGoogle, faAmazon, faSpotify, faMicrosoft } from "@fortawesome/free-brands-svg-icons";
+import "./style.css";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Grid from "@mui/material/Grid";
+import Slide from "@mui/material/Slide";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import { useNavigate } from "react-router-dom";
+import GithubIcon from "@material-ui/icons/GitHub";
+import {
+  faGoogle,
+  faAmazon,
+  faSpotify,
+  faMicrosoft,
+} from "@fortawesome/free-brands-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 export default function Landing() {
+  let navigate = useNavigate();
+ 
   return (
     <>
-    {/* navbar */}
-    <Slide direction='down' in={true} timeout={800}>
-      <AppBar position='sticky' className='AppBar' color='inherit'>
-        <AppToolbar>
-          <div className='HeaderContainer'>
-            <div className='HeaderLeftContainer'>
-              <a href="http://localhost:3000/">
-              <img src={require('../../images/logo.svg').default} alt="logo" width="32" height="32"/>
-              </a>
-            </div>
-            <div>
-              <Button color="inherit" id="login" onClick={() => (window.location.href = "http://localhost:3000/signin")}>Login</Button>
-              <Button
-                id='github-button'
-                color='inherit'
-                onClick={() => (window.location.href = 'https://github.com/roccopolimen/nextround')}
-              >
-                <GithubIcon></GithubIcon>
-              </Button>
+      {/* navbar */}
+      <Slide direction="down" in={true} timeout={800}>
+        <AppBar position="sticky" className="AppBar" color="inherit">
+          <Toolbar>
+            <div className="HeaderContainer">
+              <div className="HeaderLeftContainer">
+                <a href="http://localhost:3000/">
+                  <img
+                    src={require("../../images/logo.svg").default}
+                    alt="logo"
+                    width="32"
+                    height="32"
+                  />
+                </a>
+              </div>
+              <div>
+                <Button
+                  color="inherit"
+                  id="login"
+                  onClick={() => navigate('../signin', {replace: true})}
+                >
+                  Login
+                </Button>
+                <Button
+                  id="github-button"
+                  color="inherit"
+                  onClick={() =>
+                    (window.location.href =
+                      "https://github.com/roccopolimen/nextround")
+                  }
```

Comment:
Should I remove this then because this is a direct redirect, not a single page change

Reply:
This is good, I think